src/__stories__/useMediatedState.story.tsx

```diff
@@ -0,0 +1,28 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import { useMediatedState } from '../useMediatedState';
+import ShowDocs from './util/ShowDocs';
+
+const InputMediator = s => s.replace(/[\s]+/, ' ');
```

Comment:
I think the regex needs to be global and only replace double spaces?

```js
const InputMediator = s => s.replace(/\s\s+/g, ' ');
```

Reply:
Typo%) recently been writing in php - always forget to puth the `g` flag in JS's regexes..

Reply:
Doesn't the regex have to be `/\s\s+/g` to only replace double spaces?

I would also change `InputMediator` to `inputMediator` as it's the convention in JS.

Reply:
There can be not only space sharacters (4ex tab character) so have to replace them too.
Anyways not sure if it is important for the demo.

Reply:
Doesn't `\s` on its own cover all whitespace characters? I don't think you need the square brackets.

It's not super important for the demo but best to give a common example that is correct/